src/skins/vector/css/matrix-react-sdk/structures/_GroupView.scss

```diff
@@ -304,6 +304,18 @@ limitations under the License.
 }
 
 .mx_GroupView_groupDesc textarea {
-    width: 50%;
+    width: 100%;
+    max-width: 100%;
```

Comment:
shouldn't be needed?

Reply:
It just stops the user being silly and dragging the textarea wider than it should go. But it still allows for them to adjust its vertical height.

Reply:
"it's textareas being weird" i think is the answer then :D